### ISSUE TITLE
I can not use your code to train VOCdataset 

### ISSUE THREAD

**Issue by yuyijie1995** [andy-yun/pytorch-0.4-yolov3#27]
I use the https://github.com/marvis/pytorch-yolo2 method and get the voc_train.txt and I use the voc.data 
the command like python train.py -d cfg/voc.data -c cfg/yolo_v3.cfg -w yolov3.weights
Then I modify the some value to adapt the original voc yolo_layer
 [yolo]
mask = 0,1,2
anchors = 10,13,  16,30,  33,23,  30,61,  62,45,  59,119,  116,90,  156,198,  373,326
classes=20
1.I modify the classes number to 20
[convolutional]
size=1
stride=1
pad=1
filters=75
activation=linear
2.I changed the three conv layer's filter numbers =3(20+5)=75
in addition to these operations 
I changed the batchsize and sub
batch=64
subdivisions=16
# Training
;batch=64
;subdivisions=16
BUT! The strange thing happend 
![image](https://user-images.githubusercontent.com/30818917/46736743-5ec28080-cccc-11e8-96e6-9163a9ab1996.png)
The training stopped
